src/DatePicker/style.css

```diff
@@ -0,0 +1,200 @@
+@import '../variable.css';
+
+.ant-picker {
+  @apply h-10 px-4 border-transparent shadow-none rounded-lg;
+  @apply hover:border-neutral-300 focus:border-neutral-300;
+
+  &-panel-container {
+    @apply rounded-lg;
+  }
+
+  &-input > input {
+    @apply text-neutral-700 font-inter;
+  }
+
+  &-header {
+    @apply border-none;
+  }
+
+  &-range {
+    .ant-picker-active-bar {
+      @apply bg-slate-300;
+    }
+  }
+
+  &-range-separator {
+    @apply px-4 text-slate-300;
+  }
+
+  &-suffix {
+    @apply text-slate-700;
+  }
+
+  &-clear > span > svg {
+    @apply h-4 w-4 text-transparent;
+    background-image: var(--ic-x-16-neutral);
+  }
+
+  &-cell-in-view.ant-picker-cell-today {
+    .ant-picker-cell-inner::before {
+      @apply rounded-full border-neutral-300;
+    }
+  }
+
+  &-cell .ant-picker-cell-inner {
+    @apply rounded-full inline-block w-auto;
+    @apply px-0;
+  }
+
+  &-year-panel .ant-picker-cell-inner {
+    @apply px-2.5 rounded-lg;
+  }
+
+  &-cell-in-view.ant-picker-cell-selected {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single)::before,
+  &-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single)::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range {
+    @apply text-white;
+  }
+
+  &-header-view button {
+    @apply text-slate-700 text-base font-inter font-bold uppercase leading-6;
+    @apply hover:opacity-80 hover:text-slate-700;
+  }
+
+  &-header-view button:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range-end {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range {
+    @apply hover:bg-slate-100;
+
+    .ant-picker-cell-range-hover {
+      @apply hover:bg-red-300;
+      @apply bg-pink-600;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range.ant-picker-cell-range-hover::before {
+    @apply bg-slate-400;
+    @apply hover:bg-pink-200;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single).ant-picker-cell-range-hover-start::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single):not(.ant-picker-cell-range-end)
+    .ant-picker-cell-inner {
+    @apply rounded-tl-2xl rounded-bl-2xl;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single):not(.ant-picker-cell-range-start)
+    .ant-picker-cell-inner {
+    @apply rounded-tr-2xl rounded-br-2xl;
+  }
+}
+
+.ant-form-item-label {
+  @apply pb-2 mb-0;
+}
+
+.ant-form-item .ant-form-item-label {
+  @apply leading-none;
+}
```

Comment:
We shouldn't have these rules here. If it is a general rule, you must add it in form item styles file. If it is this component specifically, you can add it here by referencing the component class.